.github/workflows/release_image.yaml

```diff
@@ -24,7 +25,11 @@ jobs:
         fetch-tags: true
     - name: Set up Docker Buildx
       uses: docker/setup-buildx-action@v3
+    - name: Install build dependencies
+      run: sudo apt-get install -y libgpgme-dev
 
     - run: |
         docker login -u="${{ secrets.QUAY_USERNAME }}" -p="${{ secrets.QUAY_TOKEN }}" quay.io
         make docker-buildx IMG=${{ env.image_tag }}
+        make build-bundle-image BUNDLE_IMG=${{ env.bundle_image_tag }}
```

Comment:
There is an option we could use here called USE_IMAGE_DIGESTS which converts the images to SHA digest format to support offline/airgapped testing. Not required at this time I think but possibly useful in the future